packages/typespec-ts/src/modular/helpers/namingHelpers.ts

```diff
@@ -77,6 +76,10 @@ export function getClassicalLayerPrefix(
   return prefix.join(separator);
 }
 
+export function isDefined<T>(thing: T | undefined | null): thing is T {
```

Comment:
do we still need this? I remember Daniel's pr introduced this.

Reply:
I see it is still used in other places

Reply:
such as in `getModularModelFilePath`

Reply:
Oh right, I remember I've changed similar thing in modular models pr.